testEnvironment.js

```diff
@@ -13,7 +13,9 @@ class JSDOMEnvironment {
     // lazy require
     const {JSDOM} = require('jsdom');
 
-    this.document = new JSDOM(undefined, {
+    const html = '<!DOCTYPE HTML><html lang="en"><head><title>Test Environment</title></head><body></body></html>'
```

Comment:
Oh, forgot about that! I think we only need the doctype, no?

Reply:
I think that would suffice! Maybe remove the <title></title>

Reply:
`new JSDOM('<!doctype html>, {` should be good :)